src/config.ts

```diff
@@ -3035,4 +3041,39 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/fetchai/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.provenance.io/',
+		rest: 'https://api.provenance.io',
+		chainId: 'pio-mainnet-1',
+		chainName: 'provenance',
+		stakeCurrency: {
+			coinDenom: 'hash',
+			coinMinimalDenom: 'nhash',
+			coinDecimals: 9,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/hash.svg',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('pb'), // Please check
+		currencies: [
+			{
+				coinDenom: 'hash',
+				coinMinimalDenom: 'nhash',
+				coinDecimals: 9,
+				coinImageUrl: window.location.origin + '/public/assets/tokens/hash.svg',
+			},
+		],
+		feeCurrencies: [
+			{
+				coinDenom: 'hash',
+				coinMinimalDenom: 'nhash',
+				coinDecimals: 9,
+				coinImageUrl: window.location.origin + '/public/assets/tokens/hash.svg',
+			},
+		],
+		coinType: 118,
```

Comment:
don't need CoinType this here since it's already specified above.
Can we please add the gas price step?